tests/csapi/apidoc_room_receipts_test.go

```diff
@@ -5,56 +5,72 @@ import (
 
 	"github.com/matrix-org/complement/internal/b"
 	"github.com/matrix-org/complement/internal/client"
+	"github.com/matrix-org/complement/internal/docker"
 	"github.com/tidwall/gjson"
 )
 
 // tests/10apidoc/37room-receipts.pl
 
+func createRoomForReadReceipts(t *testing.T, c *client.CSAPI, deployment *docker.Deployment) (string, string) {
+	roomID := c.CreateRoom(t, map[string]interface{}{"preset": "public_chat"})
+
+	c.MustSyncUntil(t, client.SyncReq{}, client.SyncJoinedTo(c.UserID, roomID))
+
+	eventID := c.SendEventSynced(t, roomID, b.Event{
+		Type: "m.room.message",
+		Content: map[string]interface{}{
+			"msgtype": "m.text",
+			"body":    "Hello world!",
+		},
+	})
+
+	return roomID, eventID
+}
+
+func syncHasReadReceipt(roomID string) client.SyncCheckOpt {
```

Comment:
It would be nice if this took an event ID to check which event you want a receipt for.